docs/Decorators.md

```diff
@@ -57,7 +57,7 @@ Note: using an arrow function will break the binding of `this` to the Fastify `r
 <a name="decorators-encapsulation"></a>
 #### Decorators and encapsulation
 
-Decorators are not *overwritable* within the same encapsulated plugin. If you try to declare a decorator that was previously declared *(in other words, use the same name)*, `decorate`, `decorateRequest` and `decorateReply` will throw an exception.
+Decorators are not *overridable* within the same encapsulated plugin. If you try to declare a decorator that was previously declared *(in other words, use the same name)*, `decorate`, `decorateRequest` and `decorateReply` will throw an exception.
```

Comment:
The whole sentence sounds really complicated. What about?

If you define a decorator with the same name more than once in the same plugin, fastify will throw an exception.

Reply:
Much better indeed

Reply:
The whole sentence would be:

> If you define a decorator (`decorate`, `decorateRequest` and `decorateReply`) with the same name more than once in the same plugin fastify will throw an exception.

Reply:
In the same *encapsulated* plugin. Should we emphasise that?

Reply:
Yes